api/base/settings/__init__.py

```diff
@@ -11,19 +11,27 @@
 import itertools
 
 from .defaults import *  # noqa
+from website import settings
 
 try:
     from .local import *  # noqa
 except ImportError as error:
     warnings.warn('No api/base/settings/local.py settings file found. Did you remember to '
                   'copy local-dist.py to local.py?', ImportWarning)
 
+if not settings.DEV_MODE:
+    if not local.BYPASS_THROTTLE_TOKEN:
```

Comment:
Does this work? `*` is imported from `.local` above, but `local` is imported below. I'd expect it to be undefined. Ditto for `defaults`